src/main/java/org/blyznytsia/exception/BeanConfigurationException.java

```diff
@@ -0,0 +1,15 @@
+package org.blyznytsia.exception;
+
+import org.blyznytsia.context.ApplicationContext;
+
+/**
+ * Exception thrown when the validation of a bean definition failed.
+ *
+ * @see org.blyznytsia.bpp.BeanPostProcessor#configure(Object, ApplicationContext)
+ */
+public class BeanConfigurationException extends RuntimeException {
+
+  public BeanConfigurationException(String message, Throwable cause) {
```

Comment:
Maybe, additionally for this one constructor we need to add one more constructor only with message parameter to call constructor of super class?